### DISCUSSION TITLE
How I can disable some command in the search result

### DISCUSSION THREAD

**Discussion by YerongLi** [davatorium/rofi#2013] [Q&A]
In RoFi when I search for "Firefox", I want to disable some search results. 
For example, when "Firefox (Web Browser)" and "Firefox-debug" appears I want to disable the "Firefox-debug", are there any ways to do this?
![os_screenshot](https://github.com/user-attachments/assets/e27b34eb-045d-46fd-9ec6-d9d3f2cea699)

**Comment by DaveDavenport** [comment]
rofi follows the xdg standard , so use one of the methods described in the specification. This should then generally apply to all xdg based launchers.

I personally copy the relevant desktop file in `~/.local/share/applications/` and add `Hidden=true`  to it.

**Reply by YerongLi** [reply]
My Rofi seems picking `items` from PATH

**Reply by DaveDavenport** [reply]
run mode uses PATH,   drun mode uses desktop files.
